src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -339,11 +338,7 @@ private void configureJdkSourceAndTarget(Project project, Map<String, Object> pr
   }
 
   private static String getProjectKey(Project project) {
-    // SonarQube uses project keys in URL parameters without internally URL-encoding them.
-    // According to my manual tests with sonar-gradle plugin based on Sonar Runner 2.0 and Sonar 3.4.1,
-    // the current defaults will only cause a problem if project.group or project.name of
-    // the Gradle project to which the plugin is applied contains special characters.
-    // (':' works, ' ' doesn't.) In such a case, sonar.projectKey can be overridden manually.
+    String path = project.getPath();
     String name = project.getName();
     String group = project.getGroup().toString();
     return group.isEmpty() ? name : (group + ":" + name);
```

Comment:
nevermind, it's only for root project. I would rename the method and remove the useless path var.